src/bundle/Resources/config/services/tabs/locationview.yml

```diff
@@ -1,11 +1,24 @@
 services:
     EzSystems\EzPlatformAdminUi\Tab\LocationView\:
         resource: "../../../lib/Tab/LocationView/*"
+        exclude: '../../../lib/Tab/LocationView/PagerLocationToDataMapper.php'
         parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
         public: true
         tags:
             - { name: ezplatform.tab, group: location-view }
 
+    EzSystems\EzPlatformAdminUi\Tab\LocationView\PagerLocationToDataMapper:
+        autowire: true
+        public: false
+
+    EzSystems\EzPlatformAdminUi\Tab\LocationView\LocationsTab:
+        parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
+        public: true
+        arguments:
+            $defaultPaginationLimit: '$pagination.location_limit$'
```

Comment:
Do not use dynamic settings because they vanished from the product in 3.0 I believe. Inject `ConfigResolver` and use `getParameter` method to retrieve the value.